clusters/hive/pools/msp/install-config-aws-us-east-2_secret.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: install-config-aws-us-east-2
+  namespace: msp-cluster-pool
+stringData:
+  # The public key is taken from https://github.com/openshift/release/blob/80d88fb8ab7eec427d001ed7fb67741251c49dcf/core-services/ci-secret-bootstrap/_config.yaml#L914-L916
+  install-config.yaml: |
+    apiVersion: v1
+    baseDomain: hive.aws.ci.openshift.org
```

Comment:
You likely need to set up your domain, not ours

Reply:
https://docs.openshift.com/container-platform/4.8/installing/installing_aws/installing-aws-account.html#installing-aws-account

Reply:
ah, I see your other config has this right

Reply:
Hi @petr-muller I have updated the baseDomain with the MPS base domain.